tests/lapi/lib.lua

```diff
@@ -26,6 +26,22 @@ local function version()
     return t[1] or t[1/0] or t[f()==f()]
 end
 
+local function MAX_INT()
+    local max_integer = 2^51
+    if version() ~= "LuaJIT" then
```

Comment:
It should be done with `_VERSION` check.
`math.maxinteger`, `math.mininteger` are introduced in Lua 5.3.

Reply:
Fixed.